docs/README-submodule-api.md

```diff
@@ -124,6 +124,7 @@ Parameters:
   after `@` sign, in which case we check if the sums match. For other orchestrator sub-modules
```

Comment:
Nit: Two `u's` in `suuport, and one `p`.

Reply:
oh, sorry for that, and thank youuu :)